README.md

```diff
@@ -155,16 +159,16 @@ python3 -m gprofiler [options]
 gProfiler invokes `perf` in system wide mode, collecting profiling data for all running processes.
 Alongside `perf`, gProfiler invokes runtime-specific profilers for processes based on these environments:
 * Java runtimes (version 7+) based on the HotSpot JVM, including the Oracle JDK and other builds of OpenJDK like AdoptOpenJDK and Azul Zulu.
-  * Uses async-profiler.
+  * Uses async-profiler (supports x86_64 and Aarch64)
 * The CPython interpreter, versions 2.7 and 3.5-3.9.
-  * eBPF profiling (based on PyPerf) requires Linux 4.14 or higher; see [Python profiling options](#python-profiling-options) for more info.
-  * If eBPF is not available for whatever reason, py-spy is used.
+  * eBPF profiling (based on PyPerf) requires Linux 4.14 or higher; see [Python profiling options](#python-profiling-options) for more info (currently supports only x86_64).
+  * If eBPF is not available for whatever reason, py-spy is used (supports x86_64 and Aarch64).
 * PHP (Zend Engine), versions 7.0-8.0.
-  * Uses [Granulate's fork](https://github.com/Granulate/phpspy/) of the phpspy project.
+  * Uses [Granulate's fork](https://github.com/Granulate/phpspy/) of the phpspy project (currently supports only x86_64).
 * Ruby versions (versions 1.9.1 to 3.0.1)
-  * Uses [Granulate's fork](https://github.com/Granulate/rbspy) of the [rbspy](https://github.com/rbspy/rbspy) profiler.
+  * Uses [Granulate's fork](https://github.com/Granulate/rbspy) of the [rbspy](https://github.com/rbspy/rbspy) profiler (currently supports only x86_64).
 * NodeJS (version >= 10 for functioning `--perf-prof`):
-  * Uses `perf inject --jit` and NodeJS's ability to generate jitdump files. See [NodeJS profiling options](#nodejs-profiling-options).
+  * Uses `perf inject --jit` and NodeJS's ability to generate jitdump files. See [NodeJS profiling options](#nodejs-profiling-options) (supports x86_64 and Aarch64).
```

Comment:
Perhaps use https://www.tablesgenerator.com/markdown_tables and generate a table for runtimes support?
For example:
| Runtime | x86_64 | Aarch64 |
|:-------:|:-----------:|:-------:|
|   PHP   |      ✅      |    ❌    |
|   Java  |      ✅      |    ✅    |
|         |             |         |

Reply:
Very good idea

Reply:
+1